website/source/docs/internals/scheduling.html.md

```diff
@@ -28,8 +28,8 @@ spelunking through the source code.
 
 There are four primary "nouns" in Nomad; jobs, nodes, allocations, and evaluations.
 Jobs are submitted by users and represent a _desired state_. A job is a declarative description
-of tasks to run which are bounded by constraints and require resources. Nodes are the servers
-in the clusters that tasks can be scheduled on. The mapping of tasks in a job to nodes is done
+of tasks to run which are bounded by constraints and require resources. Clients are nodes in the cluster 
```

Comment:
Reword to: `Tasks can be schedule on nodes in the cluster running the Nomad Client`